elliptic-curve/Cargo.toml

```diff
@@ -47,6 +48,7 @@ arithmetic = ["ff", "group"]
 bits = ["arithmetic", "ff/bits"]
 dev = ["arithmetic", "hex-literal", "pem", "pkcs8"]
 ecdh = ["arithmetic"]
+hashing = ["digest"]
```

Comment:
Elsewhere in RustCrypto we generally just keep `digest`-related functionality gated behind the `digest` feature